src/wh_12_3/SumOfNumbers.java

```diff
@@ -0,0 +1,71 @@
+package wh_12_3;
+
+import java.util.*;
+
+public class SumOfNumbers {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        List<Double> numbers = new ArrayList<>();
+        fillListFromUser(numbers);
+        try {
+            reversOrder(numbers);
+            sumOfNumbers(numbers);
+            maxAndMinNumbers(numbers);
+        } catch (IllegalArgumentException ex) {
+            System.out.println("Na pustej liscie nie da sie wykonac operacji");
+        }
+
+    }
+
+    static void fillListFromUser(List<Double> list) {
```

Comment:
metoda powinna zwrocic liste zamiast wypelniac liste z argumentu

czemu zrobiles ta metode rekurencyjnie? java nei wspiera rekurencji ogonowej wiec nic nie zyskuejsz a aryzykujesz `StackOVerfloweException`

Reply:
dodalem bo wydalo mi sie to rozwiazanie ciekawe, w tej sytuacji - wyszedlem z zalozenia ze nie bedziemy wprowadzac duzo linijek.
Dodalem rozwiazanie z metoda zwracajaca liste.